README.md

```diff
@@ -55,6 +55,7 @@ FAUCET_CREDIT_AMOUNT_TKN  Send this amount of TKN to a user requesting TKN. TKN
 FAUCET_REFILL_FACTOR      Send factor times credit amount on refilling. Defauls to 8.
 FAUCET_REFILL_THRESHOLD   Refill when balance gets below factor times credit amount.
                           Defaults to 20.
+FAUCET_FILL_TX_WAIT       Wait for fill next faucet transactions. Defaults to 2000ms.
```

Comment:
This can be removed (see below)

Reply:
What if Ethereum ganache needs some different value?

Reply:
`await post.blockInfo.waitFor(info => info.state === BcpTransactionState.InBlock);` takes care of waiting for blocks. The nonce index problem is tendermint-specific and 50ms is fine for every slow system where Tendermint runs.

Reply:
:+1: